python/hail/docs/scripts/setupTutorial.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+###################################################################
+# Downloads tutorial files if no directory is given
+# Makes symlinks if necessary
+###################################################################
+
+set -e
+
+tutorialFiles=$1
+
+if [ -d 1kg.vds/ ]; then
+    rm -r 1kg.vds
+fi
+
+if [ -z "$tutorialFiles" ]; then
+    wget https://storage.googleapis.com/hail-tutorial/Hail_Tutorial_Data-v2.tgz && tar -xvzf Hail_Tutorial_Data-v2.tgz --strip 1
+else
+    for f in "${tutorialFiles}/*"; do
```

Comment:
Actually sorry I'm pretty sure this one can't have double quotes otherwise you won't get the glob-matching of `*`.